pkg/util/runtime/features.go

```diff
@@ -95,10 +95,10 @@ var (
 	// In each of these, keep the feature sorted by descending maturity then alphabetical
```

Comment:
In the "To promote a feature from alpha->beta" section we should add "* add to `BETA_FEATURE_GATES` in build/Makefile"

And in the "To promote a feature from beta->GA:" add "* remove from `BETA_FEATURE_GATES` in build/Makefile"